fiscalsim_us/parameters/gov/states/mn/tax/income/amt/standard_deduct.yaml

```diff
@@ -0,0 +1,21 @@
+description: MN alternative minimum tax standard deduction phase out
```

Comment:
@mitchellpound. This parameter is misnamed. It should be `std_deduct_phase_out.yaml`, and the current `std_deduct_phase_out.yaml` should be called `standard_deduct.yaml` (or just leave the old `fractional_income_threshold.yaml` parameter).